Frontend-v1-Original/components/ssBribeCreate/ssBribeCreate.tsx

```diff
@@ -381,6 +301,183 @@ export default function ssBribeCreate() {
+    setOpen(false);
+    onSelect(gauge);
+  };
+
+  const filteredGaugeOptions = useMemo(() => {
+    const go = gaugeOptions.filter((gauge) => {
+      if (search && search !== "") {
+        return (
+          gauge.address.toLowerCase().includes(search.toLowerCase()) ||
+          gauge.token0.symbol.toLowerCase().includes(search.toLowerCase()) ||
+          gauge.token1.symbol.toLowerCase().includes(search.toLowerCase())
+        );
+      } else {
+        return true;
+      }
+    });
+
+    return go;
+  }, [search, gaugeOptions]);
+
+  return (
+    <>
+      <div
+        className="relative mb-1 min-h-[100px] cursor-pointer rounded-lg border border-slate-500 p-3 transition-colors hover:border-slate-400"
+        onClick={() => {
+          openSearch();
+        }}
+      >
+        <div className="flex w-[calc(100%-24px)] items-center py-2 px-3">
+          <div className="relative box-content flex h-20 w-36 p-3">
+            <img
+              className="absolute left-0 top-3 h-16 rounded-full border-4 border-[#212b48]"
+              alt=""
+              src={value && value.token0 ? `${value.token0.logoURI}` : ""}
+              height="70px"
+              onError={(e) => {
+                (e.target as HTMLImageElement).onerror = null;
+                (e.target as HTMLImageElement).src = "/tokens/unknown-logo.png";
+              }}
+            />
+            <img
+              className="absolute left-14 top-3 z-10 h-16 rounded-full border-4 border-[#212b48]"
+              alt=""
+              src={value && value.token1 ? `${value.token1.logoURI}` : ""}
+              height="70px"
+              onError={(e) => {
+                (e.target as HTMLImageElement).onerror = null;
+                (e.target as HTMLImageElement).src = "/tokens/unknown-logo.png";
+              }}
+            />
+          </div>
+          <div>
+            <Typography className="flex-[1] text-3xl">
+              {value ? `${value.token0.symbol}/${value.token1.symbol}` : ""}
+            </Typography>
+            <Typography color="textSecondary" className="mt-1 text-xs">
+              {value?.isStable ? "Stable Pool" : "Volatile Pool"}
+            </Typography>
+          </div>
+        </div>
+      </div>
+      <Dialog
+        onClose={onClose}
+        aria-labelledby="simple-dialog-title"
+        open={open}
+      >
+        <div className="h-[600px] overflow-y-scroll p-6">
+          <TextField
+            autoFocus
+            variant="outlined"
+            fullWidth
+            placeholder="CANTO, NOTE, 0x..."
+            value={search}
+            onChange={onSearchChanged}
+            InputProps={{
+              startAdornment: (
+                <InputAdornment position="start">
+                  <Search />
+                </InputAdornment>
+              ),
+            }}
+          />
+          <div className="mt-3 flex w-full min-w-[512px] flex-col">
+            {filteredGaugeOptions &&
```

Comment:
```suggestion
            {
```
Seems superfluous as `array` will always return `true`?